src/WebJobs.Extensions.DurableTask/Options/DurableTaskOptions.cs

```diff
@@ -55,7 +57,7 @@ public string HubName
         /// <summary>
         /// The section of configuration related to storage providers.
         /// </summary>
-        public StorageProviderOptions StorageProvider { get; set; }
+        public IDictionary<string, object> Storage { get; set; } = new Dictionary<string, object>();
```

Comment:
This is pretty generic. I suggest we update the documentation comment to describe what this property is storing. One question I have: the property name is "Storage" - does that refer to Azure Storage? If so, we should probably rename this property to `AzureStorage`.  Also, does this mean we will add properties like this for each durability provider we support (e.g. "AzureEventHubs")?

Reply:
Will add examples in the morning.

Reply:
The Redis factory shown in the example now shows how different storage providers can utilize this.